### DISCUSSION TITLE
Allow creating a link that points to an external site in menus.toml

### DISCUSSION THREAD

**Discussion by Yvand** [thuliteio/doks#327] [Q&A] [answered]
## Summary

I would like to add a link (either in docs or in main section) that redirects to an external site (for example, a link "report an issue" that redirects to my GitHub project).

### Basic example

Something like this in menus.toml:

```
[[docs]]
  name = "Report an issue"
  url = "https://github.com/Yvand/LDAPCP/issues"
  weight = 100
```

Ideally, the link would render with an icon that indicates it redirects to an external site

### Motivation

I think it makes sense from a navigation perspective. But maybe I'm using a wrong approach and menus.toml is not the right approach?

**Comment by h-enk** [comment, marked as answer]
@Yvand, the headers of the left navigation sidebar (set in `./config/_default/menus.toml`) do not contain an `<a>` element (see also line 3 of `./layouts/partials/sidebar/docs-menu.html`). You could change that — for all or just for the concerning link only.

Or, you could add the link manually (like I did for the [Doks Showcase](https://getdoks.org/showcase/gamesupport/)) — add to `./layouts/partials/sidebar/docs-menu.html` something like:

```html
<h3 class="sidebar-link"><a href="https://github.com/h-enk/doks/discussions?discussions_q=category%3A%22Show+and+tell%22">Submit to Showcase →</a></h3>
```

And customize styling to your liking.
